creator-node/src/routes/files.js

```diff
@@ -433,7 +434,7 @@ module.exports = function (app) {
         storagePath: resizeResp.dir.dirDestPath,
         type: 'dir' // TODO - replace with models enum
       }
-      await DBManager.createNewDataRecord(createDirFileQueryObj, cnodeUserUUID, models.File, transaction)
+      newDataRecord = await DBManager.createNewDataRecord(createDirFileQueryObj, cnodeUserUUID, models.File, transaction)
```

Comment:
This clock logic should be encapsulated in the middleware

Reply:
discussed offline, done